kpathsea from TeX Live 2009 and later provides a version.h header with this
definition. Use it because it causes definition conflicts with kpathsea from TeX
Live 2010.

Index: xdvik-22.84.16/texk/xdvik/xdvi.h
===================================================================
--- xdvik-22.84.16.orig/texk/xdvik/xdvi.h
+++ xdvik-22.84.16/texk/xdvik/xdvi.h
@@ -242,8 +242,7 @@ typedef unsigned long xuint32;
 #define LENGTH_OF_ULONG ((sizeof(unsigned long) * CHAR_BIT + 2) / 3 + 1 + 1)
 
 #include "kpathsea/c-dir.h" /* dirent.h, NAMLEN */
-
-extern KPSEDLL char *kpathsea_version_string;
+#include "kpathsea/version.h"
 
 #ifndef	NeedFunctionPrototypes
 #  if	__STDC__
